tap_hubspot/client.py

```diff
@@ -35,82 +40,53 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         self.__session.close()
 
-    def get_new_access_token(self):
-        # This is for the oauth server-to-server access type. From the docs:
-        # To get a new access token, your app should call the /oauth/token endpoint again with the account_credentials grant.
-        # https://legacydocs.hubspot.com/docs/methods/oauth2/get-access-and-refresh-tokens
-        if self.__refresh_token is not None:
-            return
-        
-        headers = {
-            'Content-Type': "application/x-www-form-urlencoded;charset=utf-8",
-            'authorization': 'Bearer {}'.format(self.__access_token)
-        }
-        
-        params = {
-            'grant_type': 'authorization_code',
-            'client_id': self.__client_id,
-            'client_secret': self.__client_secret,
-            'redirect_uri': self.__redirect_uri,
-            'code': self.__code
-        }
-
-        data = self.request(
-            'POST',
-            url='https://api.hubapi.com/oauth/v1/token',
-            headers=headers,
-            params=params)
-        self.__access_token = data['access_token']
-        self.__expires_at = datetime.utcnow() + \
-            timedelta(seconds=data['expires_in'] - 10) # pad by 10 seconds for clock drift
-
-        with open(self.__config_path) as file:
-            config = json.load(file)
-        config['access_token'] = data['access_token']
-        with open(self.__config_path, 'w') as file:
-            json.dump(config, file, indent=2)
-
     def refresh_access_token(self):
 
         # https://legacydocs.hubspot.com/docs/methods/oauth2/refresh-access-token
-        headers = {
-            'Content-Type': "application/x-www-form-urlencoded;charset=utf-8",
-            'authorization': 'Bearer {}'.format(self.__access_token)
+        header = {
+            'authorization': 'Bearer {}'.format(self.__access_token),
+            'Content-Type': "application/x-www-form-urlencoded;charset=utf-8"
         }
         
-        params = {
+        refresh_url = "https://api.hubapi.com/oauth/v1/token"
```

Comment:
Should this exist in endpoints.py? I'm fine with leaving it here too as I don't think this will get used elsewhere.